src/components/atoms/logo/index.js

```diff
@@ -0,0 +1,13 @@
+import React from 'react'
+
+import logo from '_images/ckl-logo.png'
+
+import styles from './styles.css'
+
+const Logo = () => (
+  <div className={styles.brand}>
+    <img className={styles.brandImage} src={logo} alt="company logo" />
```

Comment:
I think you must have an Image component.

Reply:
The image component is already created in another branch I'm waiting to merge all components to refactor this